contracts/cw-core/src/contract.rs

```diff
@@ -478,6 +481,32 @@ pub fn execute_remove_item(
     }
 }
 
+pub fn execute_update_sub_daos_list(
+    deps: DepsMut,
+    env: Env,
+    sender: Addr,
+    to_add: Vec<SubDao>,
+    to_remove: Vec<String>,
+) -> Result<Response, ContractError> {
+    if env.contract.address != sender {
+        return Err(ContractError::Unauthorized {});
+    }
+
+    for addr in to_remove {
+        let addr = deps.api.addr_validate(&addr)?;
+        SUBDAO_LIST.remove(deps.storage, &addr);
+    }
+
+    for subdao in to_add {
```

Comment:
Totally ignore this for this PR, but one cool style flourish you can do in Rust is:

```rust
for SubDao { addr, charter } in to_add {
  // ...
}
```

You may want to do this to flex in your coding interviews. ;)